packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -62,15 +63,15 @@ Mandatory children prop when icon is not provided
 */
 type BaseButtonWithoutIconProps = BaseButtonCommonProps & {
   icon?: undefined;
-  children: string;
+  children: StringChildrenType;
```

Comment:
Forgot to change the type in the actual public Button.

Reply:
ops yes. Added.